Facebooky/FormSohrtcutsSettings.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Specialized;
+using System.Windows.Forms;
+
+namespace Facebooky
+{
+	public partial class FormShortcutsSettings : Form
+	{
+		private readonly StringDictionary r_ShortcutsDickDictionary;
+
+		public FormShortcutsSettings(StringDictionary i_ShortcutsDickDictionary)
+		{
+			r_ShortcutsDickDictionary = i_ShortcutsDickDictionary;
+			InitializeComponent();
+			foreach (string key in r_ShortcutsDickDictionary.Keys)
+			{
+				listBoxShortcutsList.Items.Add(key);
+			}
+			listBoxShortcutsList.SelectedIndexChanged += listBoxShortcutsListOnSelectedIndexChanged;
+		}
+
+		private void listBoxShortcutsListOnSelectedIndexChanged(object i_Sender, EventArgs i_EventArgs)
+		{
+			if (listBoxShortcutsList.SelectedItem != null)
+			{
+				textBox1.Text = r_ShortcutsDickDictionary[listBoxShortcutsList.SelectedItem.ToString()];
```

Comment:
If you do what I told you in the previous comment the item will already be the pair.